internal/meshdetect/detector.go

```diff
@@ -211,27 +219,119 @@ func isPodInitContainerInjected(meshKind MeshKind, pod *corev1.Pod) bool {
 	return false
 }
 
+//  listAllSevices lists all services in all namespaces.
+func (d *Detector) listAllSevices(ctx context.Context, pageSize int) ([]*corev1.Service, error) {
+	serviceList := []*corev1.Service{}
+	continueToken := ""
+	for {
+		partialServiceList := &corev1.ServiceList{}
+		err := d.Client.List(ctx, partialServiceList, client.Limit(pageSize), client.Continue(continueToken))
+		if err != nil {
+			return nil, err
+		}
+
+		for i := range partialServiceList.Items {
+			serviceList = append(serviceList, &partialServiceList.Items[i])
+		}
+
+		continueToken = partialServiceList.GetContinue()
+
+		if partialServiceList.RemainingItemCount == nil || *partialServiceList.RemainingItemCount == 0 {
+			break
+		}
+	}
+
+	return serviceList, nil
+}
+
+// listAllEndpoints lists all endpoints in all namespaces.
+func (d *Detector) listAllEndpoints(ctx context.Context, pageSize int) (
+	map[client.ObjectKey]*corev1.Endpoints, error) {
+	endpointsMap := map[client.ObjectKey]*corev1.Endpoints{}
```

Comment:
I'd suggest to not include variable types in variable names. Relevant post from Dave Cheney: https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat

```suggestion
	endpoints := map[client.ObjectKey]*corev1.Endpoints{}
```

Reply:
Resolved without a change and without a comment 😿

Reply:
WDYT about addressing this one @randmonkey ?

Reply:
> Resolved without a change and without a comment crying_cat_face

Maybe I forgot to click `Comment` before resolving conversation. Sorry. 

because there are both `list` and `map` in this function, I annotated the type here to note the difference.